src/sqlparser/src/parser.rs

```diff
@@ -1872,15 +1875,36 @@ impl Parser {
     }
 
     pub fn parse_columns(&mut self) -> Result<(Vec<ColumnDef>, Vec<TableConstraint>), ParserError> {
+        let (column_refs, table_constraints, _) = self.parse_columns_inner(true)?;
+        Ok((column_refs, table_constraints))
+    }
+
+    pub fn parse_columns_with_watermark(&mut self) -> Result<ColumnsDefTuple, ParserError> {
+        self.parse_columns_inner(true)
+    }
+
+    fn parse_columns_inner(
+        &mut self,
+        with_watermark: bool,
+    ) -> Result<ColumnsDefTuple, ParserError> {
```

Comment:
When will we call this method with `with_watermark` being `false`?

Reply:
When create a table without external source